tests/test_config.py

```diff
@@ -43,6 +43,14 @@ def test_s3_checkout_bucket(self):
         Config.set_config(BucketConfig.TEST_FIXTURE)
         self.assertEquals(Config.get_s3_checkout_bucket(), os.environ["DSS_S3_CHECKOUT_BUCKET_TEST"])
 
+    def test_s3_events_bucket(self):
+        Config.set_config(BucketConfig.NORMAL)
+        self.assertEquals(Config.get_flashflood_bucket(), os.environ["DSS_FLASHFLOOD_BUCKET"])
```

Comment:
Nit: `assertEquals` is deprecated, use `assertEqual` instead. (I was under the impression that `assertEquals` would stop working entirely, but it looks like it just raises a warning after 3.2, and in theory will stop working in the future.)

Reply:
changed.